README.rst

```diff
@@ -55,6 +55,37 @@ Dev mode
 
    pip install -e .[dev]
 
+Version control of database using DVC (Data Version Control)
+------------
+
+Data Version Control (DVC) is a data management tool that is meant to be run alongside Git.
+In this project DVC is used to link changes in the code to specific versions of the database. DVC can be
+used when this project is installed in Dev mode. You can read more about DVC and how to use it `here <https://dvc.org/doc/start>`_.
+Note: Remote storage of the database currently uses google drive storage that requires a "gdrive_client_secret"
+for user access authentication.
+
+DVC usage
+^^^^^^^^^
+
+To download the database:
+
+.. code:: sh
+
+   git pull
+   dvc pull # Requires access to remote storage
```

Comment:
Shouldn't there also be a command to add the client secret? Something like
```
dvc remote modify shared_gdrive gdrive_client_secret <client_secret_here>
```
(and we tell the reader that they need to contact the developers to get the client secret).

Really the readme should be up front about data access currently being restricted and requiring an access key to be shared by the developers